book/src/backends/plonky3.md

```diff
@@ -1,3 +1,3 @@
 # Plonky3
 
-powdr partially supports [plonky3](https://github.com/Plonky3/Plonky3) with the Goldilocks field. Progress is tracked [here](https://github.com/powdr-labs/powdr/issues/1468).
+powdr partially supports [plonky3](https://github.com/Plonky3/Plonky3) with the Goldilocks, BabyBear, KoalaBear, and Mersenne31 fields. Progress is tracked [here](https://github.com/powdr-labs/powdr/issues/1468).
```

Comment:
```suggestion
powdr supports [plonky3](https://github.com/Plonky3/Plonky3) with the Goldilocks, BabyBear, KoalaBear, and Mersenne31 fields.
```

Reply:
It's still partial though, isn't it? Lookups and permutations just get ignored?